sdk/resourcemanager/apimanagement/armapimanagement/go.mod

```diff
@@ -5,13 +5,13 @@ go 1.18
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.1.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.0.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/internal v1.1.1
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/internal v1.1.2-0.20230209022835-1155bce7b3e8
```

Comment:
why is this change? Is it required to use this version to get assets repo support?

Reply:
This should be a WIP PR. The dependency will be updated after my two PR merged and two internal module released.

Reply:
replaced：github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/internal v1.1.2